frontend/pages/hosts/details/cards/Activity/UpcomingActivityFeed/UpcomingActivityFeed.tsx

```diff
@@ -49,6 +56,9 @@ const UpcomingActivityFeed = ({
     );
   }
 
+  const canCancel =
+    isGlobalAdmin || isGlobalMaintainer || isTeamMaintainerOrTeamAdmin;
```

Comment:
I don't recall off the top of my head, but do you know if when a user navigates to the host details page, do we update the current team id in context to match the team id of the host? If not, we may want to call the permissions util function directly instead of relying on context.

Reply:
ah yah not sure. i'll look at into this one.

Reply:
i'll merge this in and then look at changing this in another PR